packages/plugins/rubygem-foreman_remote_execution/rubygem-foreman_remote_execution.spec

```diff
@@ -141,32 +125,31 @@ exit 0
 %files
 %dir %{gem_instdir}
 %exclude %{gem_instdir}/.*
```

Comment:
Actually, this might be it - we are excluding .babel.rc
in ansible, that also has it, we aren't excluding it

Reply:
i tried rebuilding ansible to test if it hits the same issues but it can't rebuild because of the rex - tasks mismatch

Reply:
however it did build fine very recently